pulsar/impl_consumer.go

```diff
@@ -154,27 +155,47 @@ func (c *consumer) Unsubscribe() error {
 }
 
 func (c *consumer) Receive(ctx context.Context) (Message, error) {
+	if len(c.consumers) > 1 {
```

Comment:
In current pulsar master branch, it is already support partitionNumber == 1

Reply:
In here, when `len(c.consumers) > 1` means the consumer is a `partitionConsumer`, when `len(c.consumers) = 1` means the consumer non partition.